civis/io/_files.py

```diff
@@ -181,8 +181,13 @@ def file_to_civis(buf, name=None, api_key=None, client=None, **kwargs):
     Parameters
     ----------
     buf : file-like object or str
-        The file or other buffer that you wish to upload. Strings will be
-        treated as paths to local files to open.
+        Either a file-like object for a buffer or a string for a local file
+        path.
+        Note that if a file-like object is provided and it's not
+        an :class:`io.BufferedReader <io.BufferedReader>`
+        or :class:`io.TextIoWrapper <io.TextIoWrapper>` object,
+        the current implementation requires extra disk space
+        (which could be an issue if your file is large).
```

Comment:
I compiled the Sphinx docs locally and verified that the formatting and links would all be good:

![Screen Shot 2021-11-01 at 2 54 01 PM](https://user-images.githubusercontent.com/86482098/139858331-23647bae-a907-4ae4-850e-397b3f25129c.png)